### DISCUSSION TITLE
Getting a lot of Timeout Errors on Vercel

### DISCUSSION THREAD

**Discussion by nathanaelphilip** [vercel/next.js#34642] [Help] [answered]
Random URLs are timing out with this error: `Runtime exited with error: signal: killed Runtime.ExitError`.
This only started to happen after upgrading to Next12.1 – is anyone else running into this issue?

example paths throwing errors:
`/_next/data/xTDapJt2MW055u5n8nln9/en-us.json`
` /es-es/`

**Comment by nathanaelphilip** [comment, marked as answer]
I think it was due to an older version of web pack; I removed it and that seems to have helped.

**Reply by midoghranek** [reply]
How to do that?
